client/src/pages/private/ParticipantTableFilters.js

```diff
@@ -62,7 +62,31 @@ export const ParticipantTableFilters = ({ loading, locations }) => {
           </TextField>
         </Box>
       </Grid>
-      <CheckPermissions permittedRoles={[Role.Employer, Role.HealthAuthority]}>
+      <Grid item>
+        <Box pl={2}>
+          <TextField
+            select
+            fullWidth
+            variant='filled'
+            inputProps={{ displayEmpty: true }}
+            disabled={loading || programs.length === 1}
+            value={filter[FILTERABLE_FIELDS.PROGRAM]?.value || ''}
+            onChange={({ target }) => setFilter(FILTERABLE_FIELDS.PROGRAM, target.value)}
+            aria-label='program filter'
+          >
+            {programs.length === 1 ? (
+              <MenuItem value=''>{programs[0]}</MenuItem>
+            ) : (
+              ['Program', ...programs].map((option, index) => (
```

Comment:
When I first saw the 'Program' option, I wondered what it meant. How about 'All'?

Reply:
it's basically acting as the label/ placeholder for what filter it is as there's no actual label describing it